docs/examples/promise/01_simple_connect/index.spec.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017-2018 @polkadot/api authors & contributors
```

Comment:
Remove the tests please. The apps itself are the tests. (It just adds clutter and the whole idea behind the samples is to be lean-and-mean)